caluma/conftest.py

```diff
@@ -282,3 +282,12 @@ def factory(use_table=False, use_subform=False):
         return (form, document, questions, answers)
 
     return factory
+
+
+def sorted_snapshot(name, key):
```

Comment:
Nice! But couldn't you make this into a "proper" fixture that uses the `snapshot`, so we won't have to call it with an explicit matcher param? Could be interesting. Also this seems rather specific (handles `tasks` as a special case) so probably shouldn't be in the top-level conftest

Reply:
Ahh returning a `snapshot` instance is even nicer!

But I made it reusable because this problem probably shows up in other places as well.

Reply:
Made it a fixture that accepts a name (eg. `"tasks"`) and a `key` passed to the `sorted` builtin function (eg. `lambda x: x["slug"])